Cargo.toml

```diff
@@ -61,3 +61,13 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 homepage = "https://fuel.network/"
 license = "Apache-2.0"
 repository = "https://github.com/FuelLabs/sway"
+
```

Comment:
Nice...

I can't find any reference in the docs for this because it looks like you can only configure via `workspace-root/.config/nextest.toml` unless overriden, but if this works then I like it :)

Reply:
Yeah i'm not sure it was getting picked up like I hoped. I've now made a `nextest.toml` config file inside the tests folder of `sway-lsp`. Probably a nice place for it anyway. Thanks!